src/vcModals.cpp

```diff
@@ -766,24 +766,25 @@ void vcModals_DrawImageViewer(vcState *pProgramState)
   if (pProgramState->openModals & (1 << vcMT_ImageViewer))
     ImGui::OpenPopup(vcString::Get("sceneImageViewerTitle"));
 
-  // Use 75% of the window
-  int maxX, maxY;
-  SDL_GetWindowSize(pProgramState->pWindow, &maxX, &maxY);
-  ImGui::SetNextWindowSize(ImVec2(maxX * 0.75f, maxY * 0.75f), ImGuiCond_Always);
-
+  ImGui::SetNextWindowSizeConstraints(ImVec2(50.f, 50.f), ImVec2((float)pProgramState->settings.window.width, (float)pProgramState->settings.window.height));
+  ImGui::SetNextWindowSize(ImVec2((float)pProgramState->image.width + 25, (float)pProgramState->image.height + 50), ImGuiCond_Appearing);
```

Comment:
This is very different to what was there before, this uses the entire screen instead of just 75%, why? Also, why is there a discrepancy between the magic numbers? Minimum size is 50x50, but you add 25x50?